app/views/streams/main_stream.html.haml

```diff
@@ -44,20 +38,104 @@
         %li{data: {stream: "public"}}
           = link_to t("streams.public.title"), public_stream_path, rel: "backbone", class: "hoverable"
 
-    .col-md-6
-      #aspect_stream_container.stream_container
-        = render 'aspects/aspect_stream', stream: @stream
+      .info-bar.hidden-xs
+        #selected_aspect_contacts.section
+          .title
+            %h5.stream-title
+              = @stream.title
+          .content
+
+        - if AppConfig.settings.invitations.open?
+          .section
+            .title
+              %h5.title-header
+                = t('shared.invitations.invite_your_friends')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.